liiatools/cans_pipeline/spec/CANS_schema.yml

```diff
@@ -0,0 +1,330 @@
+        Supervision (Caregiver needs): *0_3_can_be_blank
+        Involvement with Care (Caregiver needs): *0_3_can_be_blank
+        Knowledge (Caregiver needs): *0_3_can_be_blank
+        Organisation (Caregiver needs): *0_3_can_be_blank
+        Social Resources (Caregiver needs): *0_3_can_be_blank
+        Residential Stability (Caregiver needs): *0_3_can_be_blank
+        Medical/Physical (Caregiver needs): *0_3_can_be_blank
+        Mental Health (Caregiver needs): *0_3_can_be_blank
+        Substance Use (Caregiver needs): *0_3_can_be_blank
+        Developmental (Caregiver needs): *0_3_can_be_blank
+        Safety (Caregiver needs): *0_3_can_be_blank
+        Family Stress (Caregiver needs): *0_3_can_be_blank
+        Post-traumatic Reactions (Caregiver needs): *0_3_can_be_blank
+        Marital/Partner Violence in Home (Caregiver needs): *0_3_can_be_blank
+        Family Relationship to the System (Caregiver needs): *0_3_can_be_blank
+        Legal Involvement (Caregiver needs): *0_3_can_be_blank
+        Interpersonal Problems: *0_3_can_be_blank
+        Independent Living Skills: *0_3_can_be_blank
+        Young Carer/Parenting Roles*: *0_3_can_be_blank
+        Knowledge of Needs (Young Carer/Parenting Roles*): *0_3_can_be_blank
+        Supervision (Young Carer/Parenting Roles*): *0_3_can_be_blank
+        Involvement with Care (Young Carer/Parenting Roles*): *0_3_can_be_blank
+        Organisation (Young Carer/Parenting Roles*): *0_3_can_be_blank
+        Marital/Partner Violence in the Home (Young Carer/Parenting Roles*): *0_3_can_be_blank
+        Job Functioning*: *0_3_can_be_blank
+        Career Aspirations (Job Functioning*): *0_3_can_be_blank
+        Aspirational Congruence (Job Functioning*): *0_3_can_be_blank
+        Job Market Expectations (Job Functioning*): *0_3_can_be_blank
+        Work Ethic (Job Functioning*): *0_3_can_be_blank
+        Work History (Job Functioning*): *0_3_can_be_blank
+        Time Since Last Job (Job Functioning*): *0_3_can_be_blank
+        Job Turnover (Job Functioning*): *0_3_can_be_blank
+        Job Attendance (Job Functioning*): *0_3_can_be_blank
+        Job Performance (Job Functioning*): *0_3_can_be_blank
+        Job Relations (Job Functioning*): *0_3_can_be_blank
+        Job Enjoyment (Job Functioning*): *0_3_can_be_blank
+        Customer Orientation (Job Functioning*): *0_3_can_be_blank
+        Routine (Job Functioning*): *0_3_can_be_blank
+        Skills Relevant to Aspirations (Job Functioning*): *0_3_can_be_blank
+        Digital Literacy (Job Functioning*): *0_3_can_be_blank
+        Financial Literacy (Job Functioning*): *0_3_can_be_blank
+        CV/Cover Letter (Job Functioning*): *0_3_can_be_blank
+        Interview Clothes (Job Functioning*): *0_3_can_be_blank
+        Self-Care: *0_3_can_be_blank
+        Medication Adherence: *0_3_can_be_blank
+        Intimate Relationships: *0_3_can_be_blank
+        Transportation: *0_3_can_be_blank
+    CANS_0_5:
```

Comment:
Same as above, lowercase. We also need to make sure the liia-tools-pipeline-config matches this names